restapi/api.go

```diff
@@ -65,25 +65,25 @@ func Router(db *gorm.DB, locationFinder location.LocationFinder) (http.Handler,
 		rest.GetHandler(r, "/login", api.Login)
 		rest.GetHandler(r, "/auth/callback", api.AuthCallback)
 		rest.GetHandler(r, "/client_tokens", api.ReturnClientTokens)
-
-		// TODO: move back to protected endpoints
-		rest.PostHandler(r, "/stories", api.CreateStories)
-		rest.PostHandler(r, "/story", api.CreateStoriesFormData)
-		rest.PostHandler(r, "/authors", api.CreateAuthors)
 	})
 
 	// Private API
 	r.Group(func(r chi.Router) {
 		r.Use(jwtauth.Verifier(config.GetJWTKey()))
 		r.Use(Authenticator)
 
-		// rest.PostHandler(r, "/stories", api.CreateStories)
-		// rest.PostHandler(r, "/story", api.CreateStoriesFormData)
-		// rest.PostHandler(r, "/authors", api.CreateAuthors)
+		rest.GetHandler(r, "/check_auth", api.checkAuth)
+		rest.PostHandler(r, "/stories", api.CreateStories)
+		rest.PostHandler(r, "/story", api.CreateStoriesFormData)
```

Comment:
With this change now and these endpoints being protected, how would i test these endpoints through Postman? What would I need to do to be authorized?

Reply:
You would need to hit `/api/login`, login with your blueprint email, and then grab the `jwt` and `oauthstate` cookies -> 
![image](https://user-images.githubusercontent.com/18689448/111402247-41feb400-86a1-11eb-85e3-190e5ea0e068.png)